### ISSUE TITLE
Gmail notifications/counter doesn't work

### ISSUE THREAD

**Issue by nicolafarina** [wavebox/waveboxapp#1219]
* Wavebox Version: 10.0.90.2
* Operating System & Version: Windows 10
* Account type (if applicable): Gmail

Gmail notifications don't work anymore. Icon counter too.

Already tried to reload/resync.

Thanks.

**Comment by Thomas101** [comment]
Have you checked your unread count settings? https://kb.wavebox.io/gmail-unread-counts/

**Comment by nicolafarina** [comment]
Yes, I have both in "default". No issues on this since I use Wavebox, anyway, just last days.

**Comment by nicolafarina** [comment]
Today seems all works as expected (touching nothing)...

**Comment by Thomas101** [comment]
Thanks! Sounds like a glitch from Gmail. Keep an eye on it, if it happens again let me know and we can look into it further

**Comment by nicolafarina** [comment]
Same thing yesterday and today, no popup and notifications counter on icon.

![image](https://user-images.githubusercontent.com/16633188/75864383-3035d000-5e02-11ea-8713-0f2e1a673fa9.png)

**Comment by nicolafarina** [comment]
Today seems ok. Definitely a random glitch.

**Comment by Thomas101** [comment]
From the screenshot, that looks like the `Unread First` inbox type. `Default` should have `Primary`, `Updates` etc along the top...

**Comment by nicolafarina** [comment]
I don't use "Primary", "updates", etc.. Anyway, I have now set up "unread firts" in both Gmail and Wavebox.

But, sometimes it works and always worked before...

**Comment by kcinman11358** [comment]
I have had this issue for a while now. The issue seems to be when a conversation thread vs a new conversation thread.

I have changed the settings and it is still not working. 
https://github.com/wavebox/waveboxapp/issues/1177

**Comment by kcinman11358** [comment]
@nicolafarina were the emails part of a conversation? The conversation thing has thrown me off a few times. Basically, someone replied to an email. Gmail thinks it is a conversation still (even though I always had it off)

**Comment by nicolafarina** [comment]
I have managed to "Unread/Everything else" mode and all works fine since.

**Comment by kcinman11358** [comment]
I'm now not receiving notifications for gmail. I receive them on google calendar and g chat

10.0.158.2

**Comment by nmat** [comment]
This issue seems old. I have used gmail and notifications for a while and have not had any problems. Is this still a problem for you in latest stable?

**Comment by nicolafarina** [comment]
All fine for me.
